src/Editor/Util/AssetConverter.cpp

```diff
@@ -103,33 +99,34 @@ void AssetConverter::ConvertMesh(aiMesh * aMesh, Geometry::AssetFileHandler * fi
     meshData = nullptr;
 }
 
-Video::Geometry::VertexType::StaticVertex * AssetConverter::ConvertStaticVertices(aiMesh* aMesh, Geometry::AssetFileHandler* file, unsigned int numVertices) {
+Video::Geometry::VertexType::StaticVertex * AssetConverter::ConvertStaticVertices(aiMesh* aMesh, Geometry::AssetFileHandler* file, unsigned int numVertices, glm::vec3 scale, bool flipUVs) {
     Video::Geometry::VertexType::StaticVertex * vertices = new Video::Geometry::VertexType::StaticVertex[numVertices];
 
     // Positions.
     if (aMesh->HasNormals()) {
         for (unsigned int i = 0; i < numVertices; ++i) {
             Geometry::CpyVec(vertices[i].position, aMesh->mVertices[i]);
+            vertices[i].position *= scale;
         }
     } else {
         success = false;
         errorString.append("WARNING: The model has no positions yet the user is trying to import them.\n");
-        for (unsigned int i = 0; i < numVertices; ++i) {
```

Comment:
We don't have any explicit mentioning about braces, but I would expect them to be present similarly to if statements in the style example.